src/modules/Projects/components/ProjectList/ProjectList.jsx

```diff
@@ -52,13 +81,30 @@ export default function ProjectList() {
                   <td style={{ cursor: "pointer" }}>
                     <i className="fa fa-eye mx-1" />
                     <i className="fa fa-edit mx-1" />
-                    <i className="fa fa-trash mx-1" />
+                    <i
+                      className="fa fa-trash mx-1"
+                      onClick={() => handleShow(project.id)}
```

Comment:
This action will not be available to people using screen readers as we are using non-interactive elements. We need to make this a button and provide a helper text to screen readers as it is an icon button.